types/google-apps-script/google-apps-script-tests.ts

```diff
@@ -142,6 +142,35 @@ const request: GoogleAppsScript.Data_Studio.Request<any> = {
     ],
 };
 
+// Spreadsheet Rich Text Value & Builder
+// starting with sample code from
+// https://developers.google.com/apps-script/reference/spreadsheet/rich-text-value-builder
+const richTextStyle = SpreadsheetApp.newTextStyle()
+    .setUnderline(false)
+    .build();
+const richTextValue = SpreadsheetApp.newRichTextValue()
+    .setText("foo no baz")
+    .setLinkUrl(0, 3, "https://bar.foo")
+    .setLinkUrl(7, 10, "https://abc.xyz")
```

Comment:
It is prefer to use null for 3rd argument.

Reply:
Sorry, just for curiosity, which 3rd argument? These lines were copied from the Google documentation example. setLinkUrl(I,j,null) removes an existing link.